### PR TITLE
Add extension for randomly generated palette based on Adobe Leonardo

### PR DESCRIPTION
**Pull request by douglas-pires**
## Description

Adds a new simple extension for randomly generated colors palette based on the [Adobe Leonardo Contrast Colors](https://github.com/adobe/leonardo/tree/main/packages/contrast-colors) package. The initial phase of the extension will be based on the scheme variants from [Color Scheme](https://www.npmjs.com/package/color-scheme#schemes) package.

## Screencast

https://user-images.githubusercontent.com/5932230/195249658-688e582e-f531-4e37-b7a1-cacdd19912e2.mov

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/palette-colors/src/index.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  const [type, setType] = useState("mono");
+  const [colors, setColors] = useState(generateColors(type));
+
+  useEffect(() => {
+    setColors(generateColors(type));
+  }, [type]);
+
+  return (
+    <Grid
+      inset={Grid.Inset.Large}
```
Comment (pernielsentikaer) [inline extensions/palette-colors/src/index.tsx:41]:
```suggestion
```
I would remove this to make the colors fill the grid 😁

**Review by pernielsentikaer — CHANGES_REQUESTED** [review]
Hi 👋 

Thanks for your contribution 💪 

I have now tested your extension and I have some feedback ready for you:

* Add the HEX color to the Grid.Item
* When the extension is in production mode you don't have the reload functionality, you might want to add that as an ActionPanel item.
* Cache colors so they start the same every time you open the extension again since the users might have this setting enabled

CleanShot 2022-10-11 at 06 00 41@2x


I'm looking forward to testing this extension again 🔥 

Request a new review when you are ready. Feel free to contact me here or at [Slack](https://www.raycast.com/community) if you have any questions.

File: extensions/palette-colors/src/index.tsx (hunk 1 above)
Comment (douglas-pires) [reply extensions/palette-colors/src/index.tsx]: Good input, thank you very much! 👍🏻

**Comment by douglas-pires** [comment]
@pernielsentikaer, thank you very much for your time reviewing this. I just addressed your comments. Let me know if anything else needs to be changed :)

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**